src/ValueList.php

```diff
@@ -18,6 +18,7 @@ class ValueList implements
      */
     public static function make(array $params): ValueList
     {
+        /** @psalm-suppress TooManyArguments */
         $values = new static($params);
```

Comment:
Since there isn't a constructor right now, wouldn't just `$values = new static();` suffice? (this is a question for @shadowhand)

Reply:
Indeed it would. Passing `$params` doesn't mean anything in this situation.